populateReferences.js

```diff
@@ -0,0 +1,28 @@
+// Tool to aid in testing only
+// populates the references from dictionary.json with the actual references, generates populated_dictionary.json
+// allows for easy overwrite into clinical's sample-schema.json
```

Comment:
👌 Thank you for explaining in a comment why this file exists, I was confused.

I am cautious of giving the ability for someone to generate and upload to lectern a working schema that doesn't use references, so I'll just make sure the README is clear as to the purpose of this scritp.